website/docs/runtime-flags.md

```diff
@@ -0,0 +1,69 @@
+| `--audit-channel`                              | `"audit-channel"`                               | (alpha) Channel name for publishing audit violation messages. Defaults to audit-channel.                                                                                                                                                                                                                                      |
+| `--log-stats-audit`                            | `false`                                         | (alpha) Log stats metrics for the audit run.                                                                                                                                                                                                                                                                                  |
+| `--default-create-vap-binding-for-constraints` | `false`                                         | (alpha) Create VAPBinding resource for constraint of the template containing VAP-style CEL source. Allowed values are false: do not create Validating Admission Policy Binding, true: create Validating Admission Policy Binding.                                                                                             |
+| `--default-create-vap-for-templates`           | `false`                                         | (alpha) Create VAP resource for template containing VAP-style CEL source. Allowed values are false: do not create Validating Admission Policy unless generateVAP: true is set on constraint template explicitly, true: create Validating Admission Policy unless generateVAP: false is set on constraint template explicitly. |
+| `--default-wait-for-vapb-generation`           | `30`                                            | (alpha) Wait time in seconds before generating a ValidatingAdmissionPolicyBinding after a constraint CRD is created.                                                                                                                                                                                                          |
+| `--debug-use-fake-pod`                         | `false`                                         | Use a fake pod name so the Gatekeeper executable can be run outside of Kubernetes.                                                                                                                                                                                                                                            |
+| `--enable-pub-sub`                             | `false`                                         | (alpha) Enabled pubsub to publish messages.                                                                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| `--enable-violation-export`                             | `false`                                         | (alpha) Enable exporting violations to external systems.                                                                                                                                                                                                                                                                                   |
```

Reply:
@JaydipGabani Can you provide some context for this suggestion?

Reply:
We recently renamed`--enable-pub-sub` flag to `--enable-violation-export` - https://github.com/open-policy-agent/gatekeeper/blob/982e8c30213dfbe5150b127544a5008000f5022f/pkg/controller/export/export_config_controller.go#L29.

Reply:
Great, thank you!